collector/processors/odigossqldboperationprocessor/processor.go

```diff
@@ -51,12 +59,29 @@ func (sp *DBOperationProcessor) processTraces(ctx context.Context, td ptrace.Tra
 				}
 
 				// Detect the `db.operation.name` from the query text
-				operationName := DetectSQLOperationName(dbQueryText.AsString())
+				operationName, tableName := DetectSQLOperationAndTableName(dbQueryText.AsString())
 
+				// Used to build "{operation} {table}" span name
+				spanName := ""
 				// Only set the `db.operation.name` if the detected operation name is not "UNKNOWN"
 				if operationName != OperationUnknown {
 					span.Attributes().PutStr(string(semconv.DBOperationNameKey), operationName)
+					spanName = operationName
+				}
+
+				if tableName != OperationUnknown {
+					span.Attributes().PutStr(string(semconv.DBCollectionNameKey), tableName)
+					if spanName != "" {
+						spanName = spanName + " " + tableName
+					}
+				}
+
+				// If we have a new span name, use that (but only if the current span name is
+				// our default "DB" auto-instrumentation name).
+				if spanName != "" && span.Name() == "DB" {
```

Comment:
The `DB` name is specific to go auto instrumentation.
This creates some coupling. However, I don't have a suggestion for a better solution here since overriding the span name might not be desirable in a lot of cases.

Reply:
exactly my thought, I'm assuming the other languages are able to follow the span name semconv easier? It would be nice if our default name from auto-instrumentation was something more indicative